.pipelines/publish-to-pypi.yml

```diff
@@ -0,0 +1,33 @@
+variables:
+  - group: Presidio-V2-CI
+steps:
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.8'
+  - task: Bash@3
+    displayName: 'Package Wheel'
+    inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # fail on error
+        pip install wheel
+        python setup.py bdist_wheel
+  - task: Bash@3
+    displayName: 'Install twine'
+    inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # fail on error
+        pip install twine
+  - task: TwineAuthenticate@1
+    displayName: 'Authenticate with PyPi'
+    inputs:
+
+      pythonUploadServiceConnection: Pypi-Presidio
+  - task: Bash@3
+    displayName: 'Upload Wheel to PyPi'
+    inputs:
+      targetType: 'inline'
+      script: |
+        set -eux  # fail on error
+        twine upload -r presidio --verbose --config-file $(PYPIRC_PATH) --skip-existing dist/*
```

Comment:
The pipeline is failing because `$(PYPIRC_PATH)` is empty
How can this be set? And why is it set for other branches but not this one?

Reply:
This is the release pipeline, it is not meant to be ran locally. The PYPIRC_PATH is the path to the configuration for releasing to PyPI, which is only exposed to administrator.

Reply:
It's being triggered on Azure Pipelines and failing
When I assume it shouldn't be running that step

https://dev.azure.com/csedevil/Presidio/_build/results?buildId=14665&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=dee4e6ec-be1b-5671-4678-ab293b4004c5&l=23

Reply:
You're right. We've disabled it.